### PR TITLE
PO image gallery expanded view and misc items

### PR DESCRIPTION
**Pull request by isabell-lee**
From ticket: https://trello.com/c/8StDPHYj
- If the user clicks on the image, the image gallery should change to the expanded view.

From ticket: https://trello.com/c/rBvFMMOy
- The expanded view will also primarily consist of a main image.  Unlike the default view, this main image will span the entire screen.
- The main image will still offer right and left arrows, which will have the same function of scrolling through the image set.
- Instead of displaying a magnifying glass on hover, in the expanded view the mouse should become a “+” symbol while hovering over the main image.

Misc:
Fixed up total ratings, style selector having a checkmark on current style, css stylings

### CODE REVIEW

File: client/src/Components/ProductOverview/POComponents/ImageGallery.jsx [hunk 1]
```diff
@@ -28,7 +30,21 @@ class ImageGallery extends React.Component {
...
+    }
     console.log('you want to expand the image');
+    this.setState({
+      expanded: !this.state.expanded
+    });
+    this.props.expandSlide();
+  }
+
+  zoomSlide() {
+    console.log('you want to zoom');
```
Comment (thecaitlinkinney) [inline client/src/Components/ProductOverview/POComponents/ImageGallery.jsx:44]: Love this as a log, but mind commenting it out? :)

**Review by thecaitlinkinney — COMMENTED** [review]
Hey! Just saw one console log in there - Left you an in-line comment. Everything else looks really nice!

**Comment by isabell-lee** [comment]
> Hey! Just saw one console log in there - Left you an in-line comment. Everything else looks really nice!

Commented it out!

**Review by irislavie — APPROVED** [review]
Looks good to me!

**Pull request merged**